### ISSUE TITLE
make responsive and beyond 

### ISSUE THREAD

**Issue by inglorious-ratbastard** [inglorious-ratbastard/The-Bonsall-Project#17]
This project was not intentionally built with responsiveness in mind. The intent was to build something vanilla inspired and enhanced with limited external libraries. Perhaps its about time to move forward from this simplistic concept and accept some accelerated designs. But first, I would still like to see this project built out to be as responsive possible while maintaining its original format. 

<h6>Please fork bridger and create a new branch for your Pull Request.</h6>

**Comment by Elaa257** [comment]
Hi, do you need help with this issue? If so, I would like to contribute :)

**Comment by inglorious-ratbastard** [comment]
Yes, this issue is still open to anyone who would like to contribute. If you could please post a Github pages link with your PR it would be much appreciated.

**Comment by Elaa257** [comment]
@inglorious-ratbastard I am currently working on it and will do so when I am done
